src/lib.rs

```diff
@@ -175,3 +190,50 @@ pub struct Series {
 pub struct InfluxServerError {
     pub error: String,
 }
+
+pub struct AsyncUdpDb {
+    handle: Handle,
+    my_addr: SocketAddr,
+    they_addr: SocketAddr,
```

Comment:
Should this be `my_addr` and `their_addr`, to match better? Or am I misunderstanding something, like is this standard UDP terminology, are these acronyms, etc?